test/utils.test.js

```diff
@@ -41,15 +41,15 @@ describe('utils', () => {
 
   describe('isLastLineWhitespaceOnly', () => {
     it('should return true for empty string', () => {
-      expect(isLastLineWhitespaceOnly('')).toEqual(true)
+      expect(isLastLineWhitespaceOnly('')).toEqual(false)
```

Comment:
I think isLastLineWhitespaceOnly() basically resolves the interpolation which includes   property and value.

```
`${props => props.primary && css`background: #ffffff;`}`
```
in this case, isLastLineWhitespaceOnly returns true and then interleave() provides `-styled-mixin: undefined;`.

And when interpolation is value of any property: 
```
`background: ${color}`
```
in this case, isLastLineWhitespaceOnly returns false and then interleave() provides `background: $color;`.



But what if the multi interpolations in value?

```
`background: ${box} ${color}`
```
In this case, isLastLineWhitespaceOnly now returns true and then interleave() provides `background: $box -styled-mixin: color;` that causes syntax error.
It is because quasis[1] is `' '`(between two interpolations).

So I change isLastLineWhitespaceOnly returns false when receive `' '` or `''` , then above case returns true and interleave() provides `background: $box $color`. 


Is this correct?